.github/workflows/CICD.yml

```diff
@@ -122,12 +110,28 @@ jobs:
         cargo -V
         rustc -V
 
-    - name: Build
-      uses: actions-rs/cargo@v1
+    - name: Set cargo cmd
+      run: |
+        echo "CARGO_CMD=cargo" >> $GITHUB_ENV
+        echo "CARGO_CMD=cargo" >> $Env:GITHUB_ENV
```

Comment:
Why do we need both here? Is that powershell syntax? I would prefer if we could set `shell: bash` (or is it the default anyway?) and remove this line.